### ISSUE TITLE
Herald of Agony Accuracy Discrepancy

### ISSUE THREAD

**Issue by Aparaxium** [PathOfBuildingCommunity/PathOfBuilding#4991]
### Check version

- [X] I'm running the latest version of Path of Building and I've verified this by checking the [changelog](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CHANGELOG.md)

### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### What is the expected behaviour/value?

A character with only herald of agony equipped should have a chance to miss against standard bosses, pinnacle bosses, and uber pinnacle bosses.

### What is the actual behaviour/value?

A character with only herald of agony equipped has 100% chance to hit standard bosses, pinnacle bosses, and uber pinnacle bosses. I'm having issues reproducing, however changing the enemy level seems to snapshot the accuracy, if it is removed and different boss presets are chosen it will remain the same as when it was entered manually.

### How to reproduce the issue

Select herald of agony as main skill.
Change "is enemy a boss?" to pinnacle boss in configuration.
Switch to calcs.
Make sure socket group is herald of agony.
Enable show minion stats.
Note that the accuracy is 100%.
Hover over the 100% to show that the enemy level is 1.

### Character build code

```shell
eNrlXOtv2sgW_7z8FRbSSnu1hNjjd5TsCgh5tKEhkEe7X6KJGcCNsYltILTq_37PzBgwD5vBrVZXum2VwszvvOecOTOYnP79PvKkKQkjN_DPykpVLkvEd4Ke6w_Oyg_3F0dW-e-_SqdtHA9v-_WJ69GZv0q_nbLXkkemxAO6shTjcEDixwUn9Rk4jbEfD0ngt_DXILwMemflT4FPytIL9ntuvHjneDiKPuEROSs_ubEzLEs4cojfa6zGOXCEXb8bOK8kvgyDyfisjMrS1CWzVtADzHWrfdu5Twl1_bRQ0Pm307aH5yTsxjiWIvhxVq6B6XhArtwYWGFvAnzk8vFubHdMSG8_rB2SZr9PnNidkkboxo0h9h2yn-5QbGvixe7Yc0m4xCtVPYviPoixd97u7ufNkYGAQyBaw7oHThHiS9HXA9-NiTC8HbhR4B-ktRC4MfE8WMZi2GD04vqCNjaCwOsFMz8VERmpWegW9nEjiAQ8feP2iRiy2RXDdWDViyGplm0SQkbG4soeRJBI6BIngKw_RMYhJCk7igkrQNnsFiEoIKgbr6qApmUGnbylgVZmtTgn72Ls0kBFz-R37a9s0GSUwzCNtK1s_aZBDPuMYF1oXrWXSNfvZ1bu4TxyHey18Ls7moxgU7jHrySVzDL8yV4kg2HsQ1XJorayaS_ckBQga0CxKUI2xEGURZfjHpoGK1_k7JLOCYVe-84KbeQxffBDEpFwmtpdlTx8BxKD7tsvHhEjWAlIsisd0nxJA-In4uarxVaVc8XdEOIML6FZ6eCYiBWmJQqZVq5nKTjtWSuP6y7HarYYxaandLWq26aCkCojRc8Vu9trejWX6ECnPeFQoBdr-iQczLtDl3grtG3bIviFAQ08FiFl0UnTr0XJEJO4c7WJkB7oveYUR-nymZ2f3CwOT1uUvZoJdIRA0COizWw7DL7Sdtk7jKwWjoLJajNDsqrnGsHxa1FR9-0E53gEfVKH9CbO2nZjZ2pV9-CAsmGChnIU87xdJHrmOqvFMXZez4PeQNhTTMrhFN3JeAzVgEZelI5uZB0SuamW5Qjp-9G3sOLTeZa75YkLWKGFBSy3cXEpGyTittB9eFOMIQIXFrE8iLagpoygnLMzLJyXU3uZbWbGB45JQsceBhQ8e7WDGSg_pLcF0WFoaFmWBJmZexES_9tcmP8aXEhA0-9NQpoNwjI2KXaJabk-pNn2xQQvQKs6rahq1UZIN1Xb0I0s4o2biqpt2LBtm_yn-Wupti4WFNm2q7KKDM0yFGTl0-24ZthC7rxmEJSy-9LhAOLtK4gD7duVFFvgXZcMgnLWunKkGUYecHeHgbRqPll-d7FFsrPrEkLnd0C5pPk63rsj6Eui6BzHWIoIDp3hDRTSs3I59e4Ce94LbK4wSqlOj9kdJ311PRoHYcwGG9hzIsbz2h9PYslnF5TRMJhx9crSSxB4BPtn5TickET-GvbV9bxnfzJ6oVd3_H96l7mNHLmR8_wy6ffpDSdoGofsWrZ5cdFs3F8_NhOSLmHdiRRNXiL-8qz86JJZlwo6JzF2PShTTuB5eByR3lm5j70I-LnwkmG64BsnzuHGUHBWXJSj3bxWgGxOzXcSxmAF9O9O6JJMvZbze5TiAul6oSHO4kbrTDYj3lI1cBTztZPhKVYes7nQO9lMc-hkDi2sL-xlSk5m93gino9JBI2u23cdmiP5Ib8HNEfl-MVxYONy5jnxThrDbB6scGcx4JPZxLz4ZlEnszleZYU_06t8Npv8nDg403Y-mU28LEeBzz5c2M1licrh9Am2H7rIIWlqrkebuMzINj2yhGQzvI2HJOSdYSanFlSeBSQ3cUL3ZRJnp3EKkeMrtoVleIjOZZPyu5MMG-hcTiVa26UyHJrGZLPilxGZhSyPdHGKzXAfn80xYnGQz9A_mc5mwA8UmfFLjid5-Q2H1-z8ppPZxDw_tlmwjVOIA1vL56RPoArlLuYlJqeKxhP_HApenONQQVZ8V6pNA7fHD84_xY0ZuZvZylMH8eJFYqffDubILhZ-1vUb9w2_xP3JxXZG_U5B9jCCwnuVs7WLcVqexK8I9uixIvB-juHWBf5P2RnEEfZ757T9Fjb09HjRC5-yliKSImiSL8koqs_hIHNBi-bG0WaEY9jvyOiGPjZwH9ADMXZiEt7wxwgSYZg5KulMY_Z0wRrnxSLtkT6eeHT8boI9N57Tfj81mnBFMEj7c7q-OBva_ERQXW9u-EzNixMOVMZCD-6tRAvmAYWNJYPscYTaSlXmDKas6zvepEeu_eQCZWmYh1-oQvQRC_rhxTLbNjgtBf12Cvok4EsveMEeWpB4K9vS88pinh0yrkHCPRmBsfFin3vjdl6zFoY5qiwNyIi-b8FJoQfno-PrGLxwTF1xzPShHtvJhfl7wxYnmPg8ZvQMQ29elzpIC3KmHD8uQe5PRqMAFkgbWskYRyO-TGF1MdkFHe673i91d0rjtGy0NpMMXpEQe73bfm0Q-PPu2O2RsO3BwaJ3QNTWeGyLZ5J3hZIB2zjkEcgJ7IaAAwPKqaWgL20rmOeArdAmLyHBWEHhdYS-vA8JWRQCJlJJ8hHepJ8PovbJNKQRlJF50rKelb8jXVf1iiobCvpR-Y5UXTMruq2bGrzTNBtVFN1CBp0yVdmsqIquGz_ARDhoA7mhWyZQ25amVAzDsu2KrmnAAcm2BT8VmKiopg3jgDTkiqFpyKioqqrAa5UiFVkHDpppa3ZFQ6atA62logoydRCODMU2KoptK8BBUZFcUZCiqRWujKFoMJ7oDCqrFaaxIWuaDAOIstdlzaholmlaoIKuGhXDNsyKJquGQQd0kME9YBgAUVTFlOm4YVEZSK3oMK5UVIRAY802dKWiqcBANemwbgGygixLtxkRjCBdRhVbk6k4CyZ1E4HFVC-tYpgGuEOxVQWYG4zGMhFlK-sV27RBXVuXFTpsWUCjGuB-E4HS8NKywRhDpbM61RQCocBPTdcrBminVRCC4IDHDCQzDoBEugleMmxgr1saKAZWqNRLCsRFo0Gn3JizdDAH0c8twcPg6YoJSlVUSzHB_bqiME-C9iADcCpVHinUYzTqzJPgNwRhsitgJLDVdAsEgTYWXU4WKKaCbQDXTZv9BL6qqYODZRmBaZZlgCF0rVDlbZCtWiBPQQjip1IWEDQd3Io0i7oSNAa8yaJl2RZdO6Aw-FqB12ACMEagmGVrVEeqgGKD8WAp-KwsxZA2qWfwFDt5vI6miZrsXg-dG17uhnE8jk6Oj2ezWXWM42HQJ--uR6pOMDoeAxHk3hFL7CPK9rgGf-qDVu2zPhk-9ONH7UNnemS8Na6m_c8fH47Me-yED0_1cH75z8O98zx1G_YUv02nhjI10YvV-Xo7-0L98R4aHy-_kV44un3-Kn_55ozeTPXRtuXo6lvz7uj-w1HXcm6mny7c-5qqvU379XHUv4mjhnV18eH1sXWDlY93Svft7V5u33SsaPrcunycx_5zu_OuNz50P-L5l7fzz_801bu71sND__LDw-B2Jhuo9nr1-TJ-CP3r9lXPtzoGjozWRe_r1yd9hpQPUf9VHTYHUyV4R_7X98FRfTwfz4Zfnt6a04ubmms33_qKHDzVHlsXD7OLJvUH_BjU6q_1u9rFXe2pVvs4uPin_vhaq9ce72qN62Zzdgfj17Xm7OKtXn9tErnz9nbkoqfnyJaHl62e3vk6nZ2d8bK4iMwpf8oxSnYl_o7VJhpHtgZh04t5dU8-QtjG0WwVgLFqtsKhLByrIQL8aIaJsKPpuMJpmVZA_RCAsRwXsZZW3hVOX21JK5fDG9hl2IZEtyH64lMQEzZHBxdv2CZFL1qTO-RuHJ6VMb80K0vfgmD0BdyB7CrSVEs2Nfin8IYTDgFxC4-XjSHFJt2qlTSrMY7PXdjOQnbYWWzGFPgZnGeaVQ12BiixUA24sadsd0_2TPp60TtPIsIfJXsieAwNBB1OdbgUKk1x6GK6zavldKfbgeF4fiI9fLq-e2iWamTgRlJtEgYhLkHzPhrTg8RHcGbS2pX4hcWJpCgKSt7UcbR4eg6Ky4kkVw1Ts0v8aiWhO5EQ0kvp25YdVJqulR65nidSOySSUlWq8vqQWtXTQ41JCP6LS_yCm_Sk5Yy6W__kGAD6yKVkRZxI9SP4W2Lx6ZC3E8lApevR2HMdl87Kpe-J905gC_nxPcT-gJzIVf3HH5Z8pMjyf36nDWpIwKKexH0iwWFLWvPAiom6xkKV5SNNlMefivy7FAcShma56bEPfLEn8TscenUQlf7UGWDED48S_YA8NV_60_hd4le6FMUuX1LW_eiQsUd8NxqSaF2y9DKXtN9ZP8h1mw2JL82DySYPVFHzuaA8Lim_KPIRWvfKymB-UJVmbjyU-N2-xBtLlumtoNehfCTGjX7QqS_WfFILMhFoL0Ldi9D2IvS9CGMvwtyLWHygyA4Fq0LAzVxL_k6t0yx9gipHAaUb-hm21PAmtOGWPpAZ8UqNEPchu04kWqRKkId99_1Eoo_dr7_pTvoZb9b48WidSLjfhySjNxTPmIXxucci-7wB_wS1HZbyhOa1tUphOZ2laum7w7X8Uev1IsmS2rzDWSyN5TSSAADraX1ewlBdEu14XVhjCPjuiKbdBtUgZNWG5mUqf_ma5Mv0X1uTuyKt5kW6RXourRH_dqjBS0mgc-KsCcZZy4yzsivOEuxveC3OPxFm-P9_IchaXpC5Of9-jGl9zwkvyggv2ggvygyvQLT07U31FwVqMwyb7ZW-s726H4Iu7FblKhiDE0N3FEF_woOy7Fy4SR3XH6zGklRdH-SVen0MzobznROtxEVsdGe7tKZNB4NAiAcDvHgkqyH6cet7cxp5ei2ziEMEbk-bseoOsuFpC1OtUiY-ZeMSrmXDN_2ypNFzVEq7LD3BnSM52JdeiFTzoHmht1-sIQkmMQwChQQduc-9DK3WnLY79JSdbnKO_kC0gfzPnp7u_62pSSVX1wsWz8rww42UOgMuntFPzkI556D1y_0tnkp3hsdS7WUeRXixLUj6SpC80HwH6SYZEiPLR9WDANZiMYV22bJHqTrx4gNJLjwcvUqqkEpol0pqMfeKOQH9IjKtmJJGMWlGcW8WWU97InBFvBGJDxW0YyUZReTsWX6XXjCln2sfzlgtnkVKEXlasbWgFFt52v7wiDj2ULa7lldBw4Xqjl5AnwI1UN0npjdf3GQcFr7dXi60vvQiREIpKRKIQnbqhd2KCsTdKJZJarHlqx6-yrTiZb_AmlYKO18VWRCaCKiA2j_R_AhGcquvFI6CUWBVFg_DTywX0ZJMaQWzRhxawHP04CXWCRxcKDLdW7BciMeziDmqgJ-U4mHQCntLL2KNSL4UX-R7VKqNJt6OvY2fORePqbBP2tiXQQK_7w62vrex-StoVl_dYL-KZvuLHsQno_l1VA-iaIVl746zmSe_VGcvc_7bdnbC2h52yDDweiRMa5J8E2zx1RQz_SX5Xfi1bzIuv9Ci5xMtvza5-FLJUtweYanfk7Mg2SNq_eu8wpLS32ZdkGj7XAEjBcQU8MEmiSqg2cHW0MBuCkKCcc0Qdnq8yJrT481fK_Vf359jEA==
```

**Comment by QuickStick123** [comment]
Make up your mind please this is likely related to https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues/4949 and https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues/4847
